### DISCUSSION TITLE
Action schedule blue "error" line

### DISCUSSION THREAD

**Discussion by JesperNJessen** [community/community#25643] [Actions]
<p>I have this YML file which should have the correct syntax for a schedule, but the editor shows an “error” line. Does this have any significance other than showing the tooltip?</p>
<p><img src="https://user-images.githubusercontent.com/40754881/181092328-95e5b58a-349a-457b-99da-b6b2ce073f70.png" alt="image" data-base62-sha1="qcOkiaJ29b9dr2XeEvvXnpRxLcq" width="498" height="140"></p>

**Comment by thispsj** [comment, marked as answer]
<p><a class="mention" href="/jespernjessen">@JesperNJessen</a> that’s not an error line.</p>
<p>That’s a info lint underline(which means hovering over that will show some information).</p>
<p>Error lint underlining is always yellow or red. For more information, see <a href="https://codemirror.net" rel="noopener nofollow ugc">CodeMirror</a>, which is behind GitHub’s online editor.</p>
<p>Error linting looks like this :</p>
<ol>
<li>
<p>Warnings :<br>
<img src="https://user-images.githubusercontent.com/51746608/181092330-405f7db5-843d-4215-bc34-02a866f203b2.png" alt="A Warning lint" data-base62-sha1="nGBPukR87AG9acHklncL1wQ6RsD" width="572" height="199"></p>
</li>
<li>
<p>Errors:<br>
<img src="https://user-images.githubusercontent.com/51746608/181092331-40aafb76-03b2-4e73-98ad-86d46b226d0f.png" alt="An Error Lint" data-base62-sha1="fi1FCLxZIWZ5M0Ye8TuHdxet8hl" width="546" height="198"></p>
</li>
</ol>

**Comment by JesperNJessen** [comment]
<p>Thank you, it was a bit confusing since I’m used to code editors only using those lines for errors or warnings. IMO it’s a bad idea for editors to use them for info, but what can you do.</p>
